leakcanary-android-instrumentation/src/main/java/leakcanary/FailAnnotatedTestOnLeakRunListener.kt

```diff
@@ -0,0 +1,13 @@
+package leakcanary
+
+import org.junit.runner.Description
+
+class FailAnnotatedTestOnLeakRunListener: FailTestOnLeakRunListener() {
+
+  override fun skipLeakDetectionReason(description: Description): String? {
+    return if (description.getAnnotation(FailTestOnLeak::class.java) != null)
+          null
+      else
+          "skipped leak detection"
```

Comment:
How about `"skipped leak detection"` => `"test is not annotated with @FailTestOnLeak"` ?

Reply:
Yeah sounds good! Done!